### DISCUSSION TITLE
Trilium.cc web need reinitialize every for every login

### DISCUSSION THREAD

**Discussion by yongzhengooi** [TriliumNext/Trilium#4850] [Q&A] [answered]
I recently purchased the Trillium paid hosting, but when I log in every time, it takes a lot of time to reload all the pictures, code, and others. Can the domain preload everything in the server and can be used instantly after login without waiting?

**Comment by meichthys** [comment]
Trillium.cc is not officially supported by the Trilium (or new TriliumNext) projects. This would be a support question for @sigaloid

**Comment by sigaloid** [comment, marked as answer]
To answer your question, the preload directives are set properly so that your browser handles the caching. If you use the desktop app and set up syncing it'll be more reactive though.

**Reply by yongzhengooi** [reply]
I see, so basically every first login device needs to sync for a while and I will be able to access smoothly after the browser cache right?
